harness/propertyHelper.js

```diff
@@ -139,8 +139,11 @@ function isSameValue(a, b) {
   return a === b;
 }
 
-function isWritable(obj, name, verifyProp, value) {
-  var newValue = value || "unlikelyValue";
+function isWritable(obj, name, value) {
+  var unlikelyValue = Array.isArray(obj) && name === "length" ?
```

Comment:
this breaks with `test/language/global-code/decl-lex-configurable-global.js`, because it shadows `Array`.